### PR TITLE
Ggoryl neptun f1 reporting

### PR DESCRIPTION
**Pull request by polquant**
Pull request which adds f1 score reporting to neptune and which make a fix when freeze scheduler is None

### CODE REVIEW

File: models/model.py [hunk 1]
```diff
@@ -162,24 +162,30 @@ def validation_epoch_end(self, out):
                 all_predictions[nr_head] += list(torch.argmax(predictions[nr_head], dim=1).cpu().detach().numpy())
 
         for nr_head, head_targets in enumerate(all_targets):
+            head_name = head_names[nr_head]
+            targets_for_head = all_targets[nr_head]
```
Comment (alicjak1519) [inline models/model.py:166]: Isn't `targets_for_head` the same as `head_targets` from the 164 line? :)

Comment (alicjak1519) [inline models/model.py:166]: Isn't targets_for_head the same as head_targets from the 164 line? :)

**Review by alicjak1519 — APPROVED** [review]
Good idea :)

File: models/model.py (hunk 1 above)
Comment (polquant) [reply models/model.py]: Good point! Resolved

**Review by alicjak1519 — APPROVED** [review]

**Review by majsylw — APPROVED** [review]
